include/NovelRT/Timing/StepTimer.h

```diff
@@ -13,7 +13,7 @@
 namespace NovelRT::Timing {
   class StepTimer {
   private:
-    const uint64_t _frequency;
+    uint64_t _frequency;
```

Comment:
This poses a potential codegen issue. If StepTimer is made lazy, we can put this back to being `const`.